app/models/potential_payment.rb

```diff
@@ -20,6 +21,8 @@ class PotentialPayment < ApplicationRecord
     where("amount::numeric > ?", 0)
   }
 
+  scope :manual_to_be_paid, -> { where(kind: MANUAL) }
```

Comment:
Do we want to check if the amount is greater than zero and if the publisher is suspended?